acme/acme/challenges_test.py

```diff
@@ -264,7 +264,9 @@ def test_gen_verify_cert_gen_key(self):
 
     def test_verify_bad_cert(self):
         self.assertFalse(self.response.verify_cert(
-            test_util.load_cert('cert.pem')))
+            OpenSSL.crypto.load_certificate(
```

Comment:
Pinning old versions of deps in our main tests is a bad idea because we'd miss regressions in the latest versions of our dependencies :)  We could instead write a [test farm test](https://github.com/letsencrypt/letsencrypt/tree/testfarm/tests/letstest/scripts) that pins the versions in old-but-popular OSes and runs the tests on those.